network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -40,9 +41,9 @@
 
     <div class="bg-white py-3 py-md-4 px-md-5 {{section_class}}">
       <div class="row intro mb-4">
-        <h1 class="h1-heading col-12">{{product.name}}</h1>
+        <h1 class="h1-heading col-12">{{product.title}}</h1>
         <div class="col-12 d-flex flex-column flex-md-row justify-content-between">
-          <a id="product-company-url" class="product-price pni-product-intro-large mb-2 mb-md-0" href="{{product.url}}" target="_blank">{{product.company}} {% if product.product_type == "general" %}${% endif %}{{product.price}}</a>
+          <a id="product-company-url" class="product-price pni-product-intro-large mb-2 mb-md-0" href="{{product.product_url}}" target="_blank">{{product.company}} {% if product.product_type == "general" %}${% endif %}{{product.price}}</a>
```

Comment:
`product_url` is the URL to the actual product. Whereas `product.url` is the url to this page itself.